.gitignore

```diff
@@ -29,3 +29,8 @@ runTests
 # Vim 
 ##
 .swp
+.swo
```

Comment:
`.swp` needs to be `*.swp`. That way you can remove things like `.cpp.swp`, which I also am not sure even works at all. This was fixed on the last PR though, so these changes should be removed.